packages/build-utils/src/types.ts

```diff
@@ -8,16 +9,13 @@ export interface Env {
   [name: string]: string | undefined;
 }
 
-export interface File {
+export type File = FileRef | FileFsRef | FileBlob;
+export interface FileBase {
   type: string;
   mode: number;
   contentType?: string;
   toStream: () => NodeJS.ReadableStream;
   toStreamAsync?: () => Promise<NodeJS.ReadableStream>;
-  /**
-   * The absolute path to the file in the filesystem
-   */
-  fsPath?: string;
 }
 
 export interface Files {
```

Comment:
Since `Files` is unchanged, this PR is effectively removing `Lambda` from the output however I think we often use `File` to also include `Lambda`

Reply:
"output" is defined on `BuildResultV2` and still includes `Lambda` as a possible type. `Files` has never allowed for Lambda (and is typically only used as the "input" to Builders, which makes sense that `Lambda` wouldn't be a possibility in that case).